tests/NLog.UnitTests/Targets/LogReceiverWebServiceTargetTests.cs

```diff
@@ -190,10 +190,18 @@ public void NoEmptyEventLists()
             {
                 Name = "NoEmptyEventLists_wrapper"
             };
-            asyncTarget.Initialize(configuration);
-            asyncTarget.WriteAsyncLogEvents(new[] { LogEventInfo.Create(LogLevel.Info, "logger1", "message1").WithContinuation(ex => { }) });
-            Thread.Sleep(1000);
-            Assert.Equal(1, target.SendCount);
+            try
+            {
+                asyncTarget.Initialize(configuration);
+                asyncTarget.WriteAsyncLogEvents(new[] { LogEventInfo.Create(LogLevel.Info, "logger1", "message1").WithContinuation(ex => { }) });
+                Thread.Sleep(1000);
+                Assert.Equal(1, target.SendCount);
+            }
+            finally
+            {
+                asyncTarget.Close();
```

Comment:
Idem did something like this:

https://github.com/NLog/NLog/blob/master/tests/NLog.UnitTests/NLogTestBase.cs#L62